Makefile

```diff
@@ -1,15 +1,42 @@
 default: help
 
+### help:		Show Makefile rules
+.PHONY: help
+help:
+	@echo Makefile rules:
+	@echo
+	@grep -E '^### [-A-Za-z0-9_]+:' Makefile | sed 's/###/   /'
+
+### go-lint:		Lint Go source codes
 .PHONY: lint
 lint:
 	golangci-lint run --verbose ./...
 
+### test:		Run the tests of apisix-seed
 .PHONY: test
 test:
 	ENV=test go test -race -cover -coverprofile=coverage.txt ./...
 
-.PHONY: help
-help:
-	@echo Makefile rules:
-	@echo
-	@grep -E '^### [-A-Za-z0-9_]+:' Makefile | sed 's/###/   /'
+### build:		Build apisix-seed
+.PHONY: build
+build:
+	go build
+
+### install:		Install apisix-seed
+.PHONY: install
+install:
+	install -d /usr/local/apisix-seed
+	install -d /usr/local/apisix-seed/log
+	install -d /usr/local/apisix-seed/conf
+	install apisix-seed /usr/local/apisix-seed/
```

Comment:
We should install binary to bin directory in the $PATH?

Reply:
I think It would also be nice to define like this: https://github.com/apache/apisix/blob/5b21d31c9eb5ad1d873066c8eee9c4837b8d565d/Makefile#L46-L49

Reply:
> 

I'll add `/usr/local/apisix-seed/` to $PATH

Reply:
> We should install binary to bin directory in the $PATH?

apisix-seed is not yet able to read configuration from the command line. Configuration can only depend on environment variables and the directory where the binary is located, so putting it in $PATH does not start very gracefully and can be confusing. I'll be opening a PR to fix this

Reply:
> I think It would also be nice to define like this: https://github.com/apache/apisix/blob/5b21d31c9eb5ad1d873066c8eee9c4837b8d565d/Makefile#L46-L49

done